optaplanner-docs/src/main/asciidoc/Chapter-Optimization_algorithms/Chapter-Optimization_algorithms.adoc

```diff
@@ -273,8 +269,8 @@ This is especially common during <<realTimePlanning,real-time planning>>.
 ====
 
 For synchronous termination, configure a `Termination` on a `Solver` or a `Phase` when it needs to stop.
-You can implement your own ``Termination``, but the built-in implementations should suffice for most needs.
-Every `Termination` can calculate a _time gradient_ (needed for some optimization algorithms), which is a ratio between the time already spent solving and the estimated entire solving time of the `Solver` or ``Phase``.
+The built-in implementations of these should be sufficient, however a custom `Termination` can also be used.
+Every `Termination` can calculate a _time gradient_ (needed for some optimization algorithms), which is a ratio between the time already spent solving and the estimated entire solving time of the `Solver` or Phase`.
```

Comment:
No, no new PR, just add a commit on your fork/branch and this PR will automatically be adjusted. These comments will be archived so only the open comments remain visible by default.